src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/BlockPlacer.java

```diff
@@ -104,7 +104,7 @@ private BlockDispenseHandler onBlockDispense() {
 
                 if (item != null) {
                     // Check if this Item can even be placed down
-                    if (!(item instanceof NotPlaceable)) {
+                    if (!(item instanceof NotPlaceable) && !item.isDisabled()) {
```

Comment:
This will check world-specific disablement in addition to global.

```suggestion
                    if (!(item instanceof NotPlaceable) && !item.isDisabledIn(dispenser.getWorld())) {
```